.github/workflows/build.yaml

```diff
@@ -330,29 +330,29 @@ jobs:
     steps:
       - uses: actions/download-artifact@v3
         with:
-          path: artifacts
+          name: Windows-installer
```

Comment:
A useful enhancement for `actions/download-artifact` would be to accept a list of artifact names. Part of my rationale for using its "download everything" feature was to avoid requiring changes in two places if we wanted to change the set of posted assets: I didn't want to hard-code the set of assets in the release job.

If we could download a specific list to a specific subdirectory, then we could continue passing `action-gh-release` a `files:` argument specifying everything in that subdirectory.

I understand the speedup benefit, just explaining the rationale for the previous implementation.

Reply:
If you look at how actions like action-gh-release are used in the wild it is uncommon to hard-code input parameters in _another_ action. I would encourage us to KISS here. Rather than providing useful indirection, placing artifact specifics in a separate repository increases complexity and is a bit confusing.

Reply:
To provide more details: the build.yaml workflow steps define the artifacts that are being uploaded (**Windows-metadata**, et al.) so, imo, it's a bit odd to then hard-code expectations about the artifacts in another repository. I think we want to consolidate this information so that is 1) easy to understand and 2) easy to update.